plugins/catalog/src/plugin.ts

```diff
@@ -59,3 +60,36 @@ export const CatalogEntityPage = catalogPlugin.provide(
     mountPoint: entityRouteRef,
   }),
 );
+
+export const EntityHasSystemsCard = catalogPlugin.provide(
```

Comment:
Bit hesitant about adding these here tbh, although not sure were they would go instead. This plugin can easily grow out of control xP

Reply:
Hmm, yeah resonable. I'm not sure if these tables are the final solution, but an easy thing we can do for now to get at least a full connection between the entities going.
We could create a separate ecosystem plugin, but on the other hand I'm already questioning why we have separate `org` and `api-docs` plugins.

Reply:
Yep idd, think we need a bit more info and thinking here before we can make good decisions. No matter what I think we're gonna wanna end up wanting to do some breaking changes here in a bit, so don't wanna block the PR on this discussion

Reply:
@Rugvip How do we continue here?

Reply:
Like this :grin: